extension/src/experiments/columns/collect/timestamp.ts

```diff
@@ -0,0 +1,14 @@
+import { ColumnAccumulator } from './util'
+import { ColumnType } from '../../webview/contract'
+
+export const timestampColumn = {
+  hasChildren: false,
+  label: 'Timestamp',
+  parentPath: 'timestamp',
+  path: 'timestamp:timestamp',
```

Comment:
When there is lots of `type === ColumnType.TIMESTAMP ? ` (or similar) around in the code that is a smell.

I think if you had `path: 'timestamp'` and `parentPath: ''` the rest of the code might be simpler. You could also consider making both the path and label `'Timestamp'` and whilst collecting the variable [here](https://github.com/iterative/vscode-dvc/pull/2239/files#diff-d9605ebc61e90bded81b48018d5d07110af18b48d720bec74e060bd0b8c111cbR134) you could use 

```
if (timestamp) {
    experiment.Timestamp = timestamp
}
```

FYI the reason we have the colon separated path is because of the (usually) nested data structure that looks something like this

```
{
	params: {
		params.yaml: {
			lr: value,
			acc: value
		}
	}
...
}
```

Reply:
Raised https://github.com/iterative/vscode-dvc/pull/2247 to show you want I am talking about.

Tests should also be fixed on that branch 🙂

Reply:
Makes sense and simplifies things for sure!